### ISSUE TITLE
broken link in readme.md

### ISSUE THREAD

**Issue by kweveen** [tmforum-rand/schemas#69]
see text "the design guidelines described here"  https://github.com/tmforum-rand/schemas/blob/master/dev/README.md gives 404
I'm not sure what the right link is ...

**Comment by sharrop** [comment]
You're right, I think the original schema guidelines file got deleted,  but I will dig it out and reestablish it from GitHub history.

**Comment by hrodrigues-tmforum** [comment]
Thank you Kevin.

Steve can you check if this version of the README @ https://github.com/tmforum-rand/schemas/commit/fca5378a37b4ff08a84002f573d82d60faf9e705 is the right one, if so I will add it to candidates so it will join on Pierre's merge.

**Comment by alirezayosefian** [comment]
Hi, I want to put the Eskima code in the category page, for example, in the category of Qaytaran carpet ( **[فرش قیطران](https://hyperfarsh.com/product-brands/%d9%81%d8%b1%d8%b4-%d9%82%db%8c%d8%b7%d8%b1%d8%a7%d9%86/)** ), I need to use Eskima so that the user can see our description more easily in the Google results.
I tried to do this with the yoast plugin, I also tried with the script code, but it didn't work.
Who can help me?
thank you
